example/z-docker/esy.json

```diff
@@ -9,7 +9,7 @@
   },
   "esy": {
     "build": [
-      "dune build --root . ./app.exe"
+      "dune build --root . --profile release ./app.exe"
```

Comment:
What is the purpose of this `--profile release`?

Reply:
Same as above: removed